tests/h/services/user_signup_test.py

```diff
@@ -130,6 +131,42 @@ def test_signup_records_stats_if_present(self, svc, stats):
 
         stats.incr.assert_called_once_with("auth.local.register")
 
+    def test_signup_logs_conflict_error_when_account_with_email_already_exists(
+        self, svc, db_session, patch
+    ):
+        log = patch("h.services.user_signup.log")
+
+        with pytest.raises(ConflictError):
+            svc.signup(username="foo", email="foo@bar.com")
+            svc.signup(username="foo", email="foo@bar.com")
+
+        log.warning.assert_called_once_with(
+            "email conflict error occured during user signup"
+        )
+
+    @pytest.mark.parametrize(
+        "username,email",
+        [
+            # In the real world these values would be identical to the first signup but
```

Comment:
> real world these values would be identical to the first signup 

Is this because we are assuming that this exception will only occur when two signup requests are submitted very close together and thus very likely by the same user?